src/main/scala/tech/sourced/gemini/ReportSparkApp.scala

```diff
@@ -1,34 +1,46 @@
 package tech.sourced.gemini
 
+import com.datastax.driver.core.Cluster
+
 object ReportSparkApp extends App {
   def printUsage(): Unit = {
-    println("Usage: ./report <repository>")
+    println("Usage: ./report [--detailed]")
     println("")
-    println("Finds duplicated files among hashed repositories")
-    println("  <repository> - repository url, example: github.com/src-d/go-git.git")
+    println("Finds duplicate files among hashed repositories")
+    println("  --detailed to see file paths of each duplicated file")
     System.exit(2)
   }
 
-  if (args.length <= 0) {
-    printUsage()
+  def print(report: Iterable[Any], detailed: Boolean): Unit = {
+    if (report.isEmpty) {
+      println(s"No duplicates found.")
+    } else if (detailed) {
+      report.foreach { item =>
+        val duplicateFiles = item.asInstanceOf[Iterable[RepoFile]]
+        val count = duplicateFiles.count(_ => true)
```

Comment:
can you use `duplicateFiles.size` here?

Reply:
It's great to know that `.count(_=>true)` method is not the proper way to do it.
Thanks for shaming me. I really deserved it :dagger:

Reply:
haha. no shaming man. we are learning. Hope you will point my mistakes too. :)